docker-runner/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This is the entrypoint for the zgrab2_runner container.
+# Runs the zgrab2 binary, passing along any arguments, with stdin containing the single line: the ZGRAB_TARGET.
+
+set -e
+
+cd /go/src/github.com/zmap/zgrab2
+
+if ! [ -x $ZGRAB_REBUILD ]; then
+  if ! [ -x $ZGRAB_BRANCH ]; then
+    git checkout $ZGRAB_BRANCH
+  fi
+  git pull
+  pushd cmd/zgrab2 && go build && popd
```

Comment:
@justinbastress, similar question [to the above](https://github.com/zmap/zgrab2/pull/30/files#r159444127), but it'd be nice if it's possible to use `make clean zgrab2` to keep things centralized in the `Makefile` (minor nitpick, though).